src/node_i18n.cc

```diff
@@ -62,6 +72,306 @@ bool flag_icu_data_dir = false;
 
 namespace i18n {
 
+#define STRINGLEN(isolate, buf, len)                                          \
+  String::NewFromUtf8(isolate, buf, v8::NewStringType::kNormal, len)          \
+      .ToLocalChecked()
+
+#define STRING(isolate, buf)                                                  \
+  String::NewFromUtf8(isolate, buf, v8::NewStringType::kNormal, len)          \
```

Comment:
Also, I'm not a huge fan hiding this behind macros.  I get that it's for brevity but IMO it mostly just obscures what is really happening.